server/bin/pbench-index.py

```diff
@@ -423,23 +468,29 @@ def main(options, name):
                     rename_tb_link(
                         tb, Path(controller_path, f"{linkerrdest}.1"), idxctx.logger,
                     )
-                elif tb_res in (2, 3):
+                elif tb_res.value in (2, 3):
                     assert False, (
                         f"Logic Bomb!  Unexpected tar ball handling "
-                        f"result status {tb_res:d} for tar ball {tb}"
+                        f"result status {tb_res.value:d} for tar ball {tb}"
                     )
-                elif tb_res >= 4 or res <= 11:
+                elif (
+                    tb_res.value >= 4
+                    or tb_res.value < error_code["GENERIC_ERROR"].value
+                ):
```

Comment:
This has changed the original code, regardless of the inscrutability of the original intent. The original was `tb_res >= 4 or res <= 11` and the new code checks both conditions against `tb_res`.

Reply:
I think the original code had a bug, I don't see `res` as a variable that we need to check in this context.

Reply:
Yaa I thought it's a bug so I replaced